pkg/kubelet/config/config.go

```diff
@@ -298,12 +298,12 @@ func (s *podStorage) seenSources(sources ...string) bool {
 	return s.sourcesSeen.HasAll(sources...)
 }
 
-func filterInvalidPods(pods []api.BoundPod, source string, recorder record.EventRecorder) (filtered []*api.BoundPod) {
+func filterInvalidPods(pods []api.Pod, source string, recorder record.EventRecorder) (filtered []*api.Pod) {
 	names := util.StringSet{}
 	for i := range pods {
 		pod := &pods[i]
 		var errlist []error
-		if errs := validation.ValidateBoundPod(pod); len(errs) != 0 {
+		if errs := validation.ValidatePod(pod); len(errs) != 0 {
```

Comment:
ValidatePod (at least currently) is validating only:
- "meta" field (which I believe is fine)
- ValidatePodSpec - this call is exactly the same inside both ValidatePod and ValidateBoundPod
So I think this change is fine.